pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -92,9 +95,12 @@ var _ = Describe("Pod Network", func() {
 		testMac := "12:34:56:78:9A:BC"
 		updateTestMac := "AF:B3:1F:78:2A:CA"
 		mtu = 1410
-		dummySwap = &netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: primaryPodInterfaceName}}
-		dummy = &netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Index: 1, MTU: mtu}}
+		originalPodInterfaceName = primaryPodInterfaceName
 		newPodInterfaceName = fmt.Sprintf("%s-nic", primaryPodInterfaceName)
+		dummySwap = &netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: originalPodInterfaceName}}
+		primaryPodInterfaceBeforeNameChange = &netlink.GenericLink{LinkAttrs: netlink.LinkAttrs{Name: originalPodInterfaceName}}
+		primaryPodInterfaceAfterNameChange = dummySwap
+		dummy = &netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Index: 1, MTU: mtu}}
```

Comment:
Shouldn't the dummy be removed?

Reply:
it's used elsewhere, so I didn't want to touch it - that's why I use dummySwap.